test/controllers/ControllerSpecBase.scala

```diff
@@ -16,17 +16,19 @@
 
 package controllers
 
-import uk.gov.hmrc.http.cache.client.CacheMap
 import base.SpecBase
 import controllers.actions.FakeDataRetrievalAction
+import org.joda.time.LocalDate
+import play.api.libs.json.Json
+import utils.Enumerable
 
-trait ControllerSpecBase extends SpecBase {
+trait ControllerSpecBase extends SpecBase with Enumerable.Implicits {
```

Comment:
Probably doesn't need `Enumerable.Implicits`

Reply:
@wolfendale thats been removed now. Paul is looking at moving the scaffolds into the project now.